sql/testdata/datetime

```diff
@@ -42,6 +42,99 @@ statement error duplicate key value \(a\)=\(2015-08-30 03:34:45\.34567\+00:00\)
 INSERT INTO t VALUES
   ('2015-08-29 20:34:45.34567-07:00'::timestamp, '2015-08-31'::date, '35h2s'::interval)
 
+statement ok
+SET TIME ZONE 'Europe/Rome'
+
+query TTT
+SELECT * FROM t WHERE a = '2015-08-25 05:45:45.53453 CET'::timestamp
+----
+2015-08-25 04:45:45.53453 +0000 UTC   2015-08-25 00:00:00 +0000 UTC   2h45m2.234s
+
+statement ok
+SET TIME ZONE "America/New_York"
+
+query TTT
+SELECT * FROM t WHERE a = '2015-08-24 23:45:45.53453 EST'::timestamp
```

Comment:
We should have a test in the `America/New_York` time zone for both `EST` and `EDT` (does this also control the interpretation of other "nearby" time zones like PST?), and one for another location where `EST` means something else.